KishonStClair-Fatima.js

```diff
@@ -0,0 +1,83 @@
+// Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(number) {
+  
+    console.log(number * number * number)
+}
+calculateCube(5)
+
+
+// Question 2
+//  write a function with an parameter for a string
+function isAVowel(str) {
+    // write a agrument for the length of the string and determine its a vowel
+    let count = 0
+    const vowels = "abpurtRWKO"
+    for(var i = 0; i < str.length; i++){
+    console.log(vowels.indexOf(str[i]) > -1)
+    }
+    
+  } 
+  
+  isAVowel("a")
+
+
+  // Question 3
+  function getTwoLengths(str) {
+  const name = "Hank";
+  const lastname = "Hippopopalous";
+  console.log([name.length, lastname.length])
+}
+getTwoLengths()
+
+// Question 4
+function getMultipleLengths(str) {
+  const name = "Hank";
+  const lastname = "Hippopopalous";
+  console.log(["hello".length, "what".length, "is".length, "up".length, "dude".length])
+}
+getMultipleLengths()
+
+// Part 2
+const maxOfThree = (num1, num2, num3) => {
+    return Math.max(num1, num2, num3)
+  }
+  
+  maxOfThree(6, 9, 1)
+  console.log(maxOfThree(6, 9, 1))
+
+
+//Question 2
+ function printLongestWord(str) {
+    let words = str.split("  ");
+    let longest = "";
+  
+    for (var word of words) {
+      if(word.length > longest.length) longest = word;
+    }
+    return longest;
+  }
+  printLongestWord("BoJack", "Princess", "Diane", "a", "Max", "Peanutbutter", "big", "blob")
```

Comment:
this doesn't give the right result. let's check in about these problems and talk through them